showkase/src/main/java/com/airbnb/android/showkase/ui/ShowkaseBrowserApp.kt

```diff
@@ -308,6 +314,24 @@ internal fun ShowkaseBodyContent(
     }
 }
 
+private fun getCategoryMetadataMap(
+    groupedComponentMap: Map<String, List<ShowkaseBrowserComponent>>,
+    groupedColorsMap: Map<String, List<ShowkaseBrowserColor>>,
+    groupedTypographyMap: Map<String, List<ShowkaseBrowserTypography>>,
+): Map<ShowkaseCategory, Int> {
+    val componentCount = groupedComponentMap.values.flatCount()
+    val colorsCount = groupedColorsMap.values.flatCount()
+    val typographyCount = groupedTypographyMap.values.flatCount()
+    
+    return mapOf(
+        ShowkaseCategory.COMPONENTS to componentCount,
```

Comment:
imo removing the intermediate property is more readable `ShowkaseCategory.COMPONENTS to groupedComponentMap.flatCount()`

Otherwise it is an extra step to look at what the property name is and where it is used, introducing a chance of mistakenly using it in the wrong place